index.html

```diff
@@ -128,14 +135,12 @@ <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
         <!-- About  section -->
         <section id="about-section">
             <div class="section-container">
-                <div class="about-title">
+                <div class="section-title">
                     <h2>About Me</h2>
-                    <p class="supporting-text">I can help you build a product,<br>
-                        feature or website Look through <br>
-                        some of my work and experience! If<br>
-                        you like what you see and have a <br>
-                        project you need coded,<br>
-                        don’t hesitate to contact me.
+                    <p class="supporting-text">
+                        I can help you build a product, feature or website. Look through some of my work and <br>
+                        experience! If you like what you see and have <br>a project you need coded, <br>don't hesitate
+                        to contact me.
                     </p>
                     <button class="project-btn">Get my resume</button>
                     <div class="about-img"></div>
```

Comment:
- [ ] Also in this section, Looking at the `Skill, Framwork` lists, according to the given design, they should be aligned to the center, I'll suggest you take a look at this also and update your design. :smile: :+1: :100: 

- - Figma design:
![Screenshot from 2023-05-11 23-16-55](https://github.com/RileyManda/portfolio-website/assets/107632780/99cfea28-37f8-45ed-89c5-74b374bc01de)

- - Your current design:
![Screenshot from 2023-05-11 23-17-15](https://github.com/RileyManda/portfolio-website/assets/107632780/169ef286-d59a-45c7-936b-aa94f9fb2980)